src/Stat.php

```diff
@@ -1115,10 +1118,10 @@ public static function constructEntryValues(
                 break;
 
             case "device":
-                $devtable = getTableForItemType('Computer_' . $value2);
+                $devtable = getTableForItemType('Item_' . $value2);
```

Comment:
Should this be changed in 10.0/bugfixes too ?

Reply:
this one yes